pkg/kubelet/kubelet.go

```diff
@@ -2151,6 +2151,96 @@ func (kl *Kubelet) HandlePodSyncs(pods []*v1.Pod) {
 	}
 }
 
+// HandlePodCleanups performs a series of cleanup work, including terminating
```

Comment:
Confirming that the test for this is already in `kubelet_test`?

Reply:
yes